src/redisearch_rs/trie_bencher/src/bencher.rs

```diff
@@ -266,11 +263,11 @@ fn load_c_benchmark<M: Measurement>(group: &mut BenchmarkGroup<'_, M>, contents:
 }
 
 pub fn rust_load_from_terms(keys: &[String]) -> RustTrieMap {
-    let words = keys.iter().map(|s| str2boxed_c_char(s)).collect::<Vec<_>>();
+    let words = keys.iter().map(|s| s.as_bytes()).collect::<Vec<_>>();
     rust_load(&words)
 }
 
-fn rust_load(words: &[Box<[c_char]>]) -> RustTrieMap {
+fn rust_load(words: &[&[u8]]) -> RustTrieMap {
```

Comment:
Why not `Box`ed anymore?

Reply:
We no longer have to, we can work directly with the slice.